bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Core/SystemUsers/IRequestClient.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Altinn.Authorization.ProblemDetails;
+
+namespace Altinn.Authentication.UI.Core.SystemUsers;
+
+public interface IRequestClient
+{    
+    Task<Result<VendorRequest>> GetVendorRequest(int partyId, Guid requestId);
```

Comment:
Should `cancellationToken` be added to parameters?

Reply:
ok fixed